impl/src/templates/partials/remote_crates_patch.template

```diff
@@ -1,31 +1,31 @@
-{%- if crate.raze_settings.patches %}
+{%- if crate.raze_settings.patches  %}
         patches = [
             {% for patch in crate.raze_settings.patches -%}
             "{{patch}}",
             {%- endfor %}
         ],
 {%- endif %}
-{%- if crate.raze_settings.patch_args %}
+{%- if crate.raze_settings.patch_args  %}
         patch_args = [
             {% for patch_arg in crate.raze_settings.patch_args -%}
             "{{patch_arg}}",
             {%- endfor %}
         ],
 {%- endif %}
-{%- if crate.raze_settings.patch_cmds %}
+{%- if crate.raze_settings.patch_cmds  %}
         patch_cmds = [
             {% for patch_cmd in crate.raze_settings.patch_cmds -%}
             "{{patch_cmd}}",
             {%- endfor %}
         ],
 {%- endif %}
-{%- if crate.raze_settings.patch_cmds_win %}
+{%- if crate.raze_settings.patch_cmds_win  %}
         patch_cmds_win = [
-            {% for patch_cmd in crate.raze_settings.patch_cmds_win -%}
+            {%  for patch_cmd in crate.raze_settings.patch_cmds_win -%}
```

Comment:
Shouldn't that change do nothing? Do you need those spaces? (also in other place). The spacing inside the template bracket is only to respect indentation.

Reply:
I does do nothing. I wasn't sure why you made your suggestion in #199 so I was trying to preempt that here. I'd be happy to remove all these extra spaces if you're fine with it.

Reply:
In #199, the indentation was removed to prevent it from surfacing in the result file, I was suggesting to move that indentation inside the tag to compensate and keep the code readable. Here it is not needed.

Reply:
Ok, I've cleaned up that commit to only include spacing that affects alignment.

Reply:
This precise space is still here :)

Reply:
Sorry, I thought I should leave that to align with the `{%- endfor %}` below it. This has been updated 😅